src/interfaces.ts

```diff
@@ -75,7 +75,7 @@ export interface ICoreKit {
    * The tKey instance, if initialized.
    * TKey is the core module on which this wrapper SDK sits for easy integration.
    **/
-  tKey: ThresholdKey | undefined;
+  tKey: ThresholdKey | null;
```

Comment:
`undefined` means "not initialized yet".
`null` means "intentionally assigned no value".
https://stackoverflow.com/a/5076962/9061331

I think `undefined` is more suitable here, no?